libpod/runtime.go

```diff
@@ -0,0 +1,76 @@
+package libpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libpod/ctr"
+	"github.com/kubernetes-incubator/cri-o/libpod/pod"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+var (
+	// ErrNotImplemented indicates that functionality is not yet implemented
+	ErrNotImplemented = fmt.Errorf("NOT IMPLEMENTED")
+)
+
+// Runtime is the core libpod runtime
+type Runtime struct {
```

Comment:
We also need a NewRuntime with functional options to set the runtime path, cgroup manager and so on. We should only add options for what a user would want to configure and not add storager server, etc. Those things should be hidden from the user.